docker.go

```diff
@@ -582,3 +583,14 @@ func getDigest(buildName string) (string, error) {
 	}
 	return "", errors.New("unable to fetch digest")
 }
+
+func getUniqueBuildName(repo, buildName string) string {
```

Comment:
What if build is triggered for a run without git clone?

Reply:
Moved away from this approach. We now generate a temporary tag using a random string.